wallets/src/main/java/bisq/wallets/bitcoind/BitcoindProcess.java

```diff
@@ -53,12 +52,11 @@ public BitcoindProcess(RpcConfig rpcConfig, Path dataDir) {
 
     @Override
     public ProcessConfig createProcessConfig() {
-        String networkArg = getParamForNetworkType(rpcConfig.networkType());
         int zmqPort = NetworkUtils.findFreeSystemPort();
         return new ProcessConfig(
                 "bitcoind",
                 List.of(
-                        networkArg,
+                        "-regtest",
```

Comment:
Is that correct?

Reply:
Yes, that's correct but you're right. It's confusing. I renamed it to `BitcoindRegtestProcess` and moved it into the tests module.

Reply:
Ah thanks.